# Generated by Django 4.1.7 on 2023-03-11 08:30

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('account', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='customuser',
            name='is_client',
            field=models.BooleanField(blank=True, default=False, verbose_name='client'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='is_pro',
            field=models.BooleanField(blank=True, default=False, verbose_name='specialization'),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
